recorder/src/main/resources/tracking.js

```diff
@@ -73,6 +73,7 @@ jsflight.TrackHash = function(event) {
         console.log(e);
     } finally {
         jsflight.eventId++;
+        jsflight.rebindAllHandlers();
```

Comment:
Yep, because hashchange/click can lead to dom updating. While this happen new frames can appear. We need to track their events too.